integration/commands_administration_test.go

```diff
@@ -986,3 +986,81 @@ func TestCommandsAdministrationCurrentOp(t *testing.T) {
 	_, ok := must.NotFail(doc.Get("inprog")).(*types.Array)
 	assert.True(t, ok)
 }
+
+func TestCommandsAdministrationListIndexes(t *testing.T) {
+	t.Parallel()
+
+	ctx, collection := setup.Setup(t, shareddata.DocumentsStrings)
+
+	for name, tc := range map[string]struct { //nolint:vet // for readability
+		collectionName any
+		expectedResult bson.D
+		expectedError  *mongo.CommandError
+		skip           string
+	}{
+		"existing-collection-default-indexes": {
+			collectionName: collection.Name(),
+			expectedResult: bson.D{
```

Comment:
Can we just skip comparing `firstBatch` field for now?

Reply:
We can use `ConvertDocument` helper to get `types.Document` and just check expected fields.

Reply:
Ah, great! Thanks!